pkg/mkimage-raw-efi/config.json

```diff
@@ -271,6 +279,22 @@
                 "rw"
             ]
         },
+        {
+            "destination": "/var",
+            "type": "tmpfs",
+            "source": "tmpfs",
+            "options": [
+                "rw"
+            ]
+        },
+        {
+            "destination": "/persist",
+            "type": "tmpfs",
+            "source": "tmpfs",
+            "options": [
+                "rw"
+            ]
+        },
```

Comment:
Please do not mount tmpfs to /persist, we prepare and mount persist partition/pool to /persist during installation.